.gitignore

```diff
@@ -19,3 +19,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# dev
+/build
```

Comment:
#15 handles removing the `build` directory from git so this PR shouldn't make this change.

- [ ] don't git-ignore `build` in this PR